### ISSUE TITLE
In the PVT-v2 paper, shouldn't the time complexity for SRA be [2*(h^2)*(w^2)*c]/(R^2) + hw(c^2) instead?

### ISSUE THREAD

**Issue by Phuoc-Hoan-Le** [whai362/PVT#107]
PVT-v2 paper said that the time complexity for SRA is [2*(h^2)*(w^2)*c]/(R^2) + hw(c^2)(R^2). 

But shouldn't it be [2*(h^2)*(w^2)*c]/(R^2) + hw(c^2) instead? 

I think the time complexity for the spatial reduction part is calculated wrong. It should be hw(c^2) instead of hw(c^2)(R^2)

**Comment by ovularslan** [comment]
I agree. I think the second part comes from convolution operations to K and V. Kernel size is (R x R), in channel is C, and the final dimension is (h/R) x (w/R) x C. Since it is calculated for K and V, it should be 2*hw(c^2) instead of hw(c^2)(R^2)
